### DISCUSSION TITLE
Editing CSS within an iFrame

### DISCUSSION THREAD

**Discussion by mattwinans** [webextensions/live-css-editor#81] [Q&A] [answered]
I really love this extension and use it all the time so kudos to the maintainers.

I'm wondering: is there a way for this extension to bypass the restrictions set by the cross-origin policy and allow styles within an iFrame to be changed? I thought this could be accomplished with Javascript for the local user, but I am no expert on the matter.

As an example, I would like to change any CSS within this embedded chart from Trading View... you'll notice the iframe:
https://www.lbank.info/exchange/zlda/usdt

Can this be accomplished with an extension like Live CSS?

Thanks

Matt

**Comment by webextensions** [comment, marked as answer]
@mattwinans 

**Did you try to go to "triple dots" menu > "More options" > Under "Load for frame/iframe" > Check the option "Load separate instances for each frame/iframe" ?**

Please note that if you see multiple editors and they are for the same origin, then Magic CSS would remember the styles code from the last edited floating editor. We don't show multiple sessions overwrite warning as of now (we plan to improve it in future).

Please note that when we use the "Apply styles automatically" feature (via the "Pin" icon), it may not work very well for within iframe pages for some cases depending on the site's iframe loading approach. But, **it seems to work fine for the site you have mentioned.**

**Reply by mattwinans** [reply]
Brilliant! I forgot about this option, though now remember messing around with it a while back. Thank you for the help and keep up the good work!
